### DISCUSSION TITLE
Are you talking about https://unstoppableswap.net/ ?

### DISCUSSION THREAD

**Discussion by flopi123** [UnstoppableSwap/unstoppableswap-gui#176] [General]
Hello, 
Just to make sure, are you talking about the software here : https://unstoppableswap.net/ ? 
I'd like to make sure it's 100% safe and not a scam website. 
Thanks !

**Comment by binarybaron** [comment]
Yes, that's correct. UnstoppableSwap.net is our website and contains the latest release from this repository.
